sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/ContactDirectorySteps.java

```diff
@@ -120,8 +120,31 @@ public ContactDirectorySteps(
               dataOperations.getPartialUuidFromAssociatedLink(
                   apiState.getCreatedContact().getUuid());
           webDriverHelpers.fillAndSubmitInWebElement(
-              CONTACT_DIRECTORY_DETAILED_PAGE_FILTER_INPUT, apiState.getCreatedContact().getUuid());
+              CONTACT_DIRECTORY_DETAILED_PAGE_FILTER_INPUT, contactUuid);
```

Comment:
why are we changing this?

Reply:
It is still using uuid of last created contact, I just refactored it to commonly used partial Id of whole UUID